src/main/java/com/iwaconsolti/league/manager/service/impl/BaseBallLeagueImpl.java

```diff
@@ -45,7 +51,7 @@ public Players findPlayers(int id) {
             if (player.getId() == id) {
                 return player;
             } else {
```

Comment:
Search logic in findPlayers, findTeams and findMatches methods:

In these methods, a "Player not found", "Team not found" or "Match not found" warning message is being thrown inside the for loop, which causes a warning to be logged whenever a partial search fails. This could fill the logs with unnecessary warnings.
Solution: Move the warning out of the for loop so that it is only logged if the item is not found after the full search.